### ISSUE TITLE
Concatenation not working

### ISSUE THREAD

**Issue by Eptis** [sir-dunxalot/ember-cli-concat#18]
I followed the install steps, but my files don't get concatenated. I still end up with a `vendor` and `app` file. This is the case in development and when building the app with `--env=production`. What could be the case here?

**Comment by sir-dunxalot** [comment]
Could you share the relevant content of your Brocfile please? You may or may not have skipped the steps outlined in [usage](https://github.com/sir-dunxalot/ember-cli-concat/wiki/Usage)

**Comment by Eptis** [comment]
I also followed the steps in usage and used the examples provided, both with no succes. However I seem to have a new issue with ember-cli-concat and ember-cli-mirage that it tries to concat some files that do not exist when building. Will look into it.


—
Met vriendelijke groet / Kind regards,


Dennis Tel

**Comment by sir-dunxalot** [comment]
Hmm, apologies for the issues. I'm not aware of any problems and know people using this in production. Perhaps it's a compatibility issue. Could you try adding `ember-cli-mirage` to the `after` property in the package.json file.

```
"ember-addon": {
  "before": "broccoli-asset-rev",
  "configPath": "tests/dummy/config",
  after: "ember-cli-mirage"
}
```

You will need to restart you server everytime you change the source inside `node_modules/`.

**Comment by sir-dunxalot** [comment]
I'm going to close this for now, given I can't replicate the issues but if you can or if you are still having issues, let me know.
